[LLM] New docs (#2828)
scaleway/docs-content: ai-data/llm-inference/how-to/delete-deployment.mdx

---
meta:

>   title: How to delete a LLM Inference deployment

Reviewer: ```suggestion
  title: How to delete an LLM Inference deployment
```

>   description: This page explains how to deletee a LLM Inference deployment

Reviewer: ```suggestion
  description: This page explains how to delete an LLM Inference deployment
```

content:

>   h1: How to delete a LLM Inference deployment

Reviewer: ```suggestion
  h1: How to delete an LLM Inference deployment
```

>   paragraph: This page explains how to deletee a LLM Inference deployment

Reviewer: ```suggestion
  paragraph: This page explains how to delete an LLM Inference deployment
```

tags: ai deleteing
dates:
  validation: 2024-03-06
  posted: 2024-03-06
categories:
  - ai-data
---

Once you have finished your inference tasks you can delete your deployment from the Scaleway console.

<Macro id="requirements" />

  - A Scaleway account logged into the [console](https://console.scaleway.com)
  - An [LLM Inference deployment](/ai-data/llm-inference/quickstart/)

1. Click **LLM Inference** in the **AI & Data** section of the [Scaleway console](https://console.scaleway.com) side menu. A list of your deployments displays.
2. Choose a deployment either by clicking its name or selecting **More info** from the drop-down menu represented by the icon <Icon name="more" /> to access the deployment dashboard.
3. Click the **Settings** tab of your deployment to display additional settings.

> 4. Click **Delete deployment** to delete your deployment.

Reviewer: ```suggestion
4. Click **Delete deployment**.
```
